src/urbit/app/chess.hoon

```diff
@@ -645,6 +777,26 @@
               :-  :~  :*  %give  %fact  ~[/game/(scot %da u.game-id)/updates]
                           %chess-update  !>([%draw-declined u.game-id])
                       ==
+                      ::  notify hark-store
+                      :*  %pass   /hark-store
+                          %agent  [our.bowl %hark-store]
+                          %poke   :-  %hark-action
+                                  !>  :-  %unread-count
+                                      :-  %chess
+                                      :-  /games/(scot %da u.game-id)/updates
+                                      :-  %.y  1
+                      ==
+                      :*  =/  title=(list content:hark)
+                            ~[[%text (crip "{<src.bowl>} declines your draw offer")]]
+                          =/  =bin:hark
+                            [/chess/games [%chess /games/(scot %da u.game-id)/updates]]
```

Comment:
It doesn't *really* matter, but none of these paths need `/updates`, since all we need is to differentiate some updates by specific game.

Also, it's hard to tell, but I think we're supposed to be using the first field of the bin differently. I think it's supposed to be something like `[category [desk unique-path]]`. So, for example:
```
[/result [%chess /<player>/<game id>]] ::  draws, stalemates, checkmates, resignations
[/updates [%chess /<player>/<game id>]]  ::  draw offers
[/challenges [%chess /]] ::  challenges
```
It's really not clear, though, if that's correct, and if so, what the benefit is as opposed to how we're currently doing it. I think it's fine to keep doing it as you are until `hark 2` is out and we get some actual guidance.